addons/website_sale/views/templates.xml

```diff
@@ -804,19 +804,162 @@
+                                                                t-att-id="'%s-%s' % (a.id,v.id)"
+                                                                t-att-value="'%s-%s' % (a.id,v.id)"
+                                                                t-att-checked="v.id in attrib_set"
+                                                            />
+                                                            <label
+                                                                class="form-check-label fw-normal"
+                                                                t-att-for="'%s-%s' % (a.id,v.id)"
+                                                                t-field="v.name"
+                                                            />
+                                                        </div>
+                                                    </t>
+                                                </div>
+                                            </t>
+                                        </div>
+                                        <div
+                                            t-elif="a.display_type == 'pills'"
+                                            class="btn-group-toggle my-2"
+                                            data-bs-toggle="buttons"
+                                        >
+                                            <t t-foreach="a.value_ids" t-as="v">
+                                                <div
+                                                    t-attf-class="o_wsale_variant_pills_shop
+                                                        form-check btn btn-primary mb-1 {{'active'
+                                                        if v.id in attrib_set else ''}}"
+                                                >
+                                                    <input
+                                                        type="checkbox"
+                                                        name="attribute_value"
+                                                        class="form-check-input"
+                                                        t-att-id="'%s-%s' % (a.id,v.id)"
+                                                        t-att-value="'%s-%s' % (a.id,v.id)"
+                                                        t-att-checked="v.id in attrib_set"
+                                                    />
+                                                    <label
+                                                        class="form-check-label fw-normal"
+                                                        t-att-for="'%s-%s' % (a.id,v.id)"
+                                                        t-field="v.name"
+                                                    />
+                                                </div>
+                                            </t>
```

Comment:
Please factorize `o_wsale_offcanvas` and `products_attributes`, IMO it doesn't make sense to duplicate this much code ^^